### PR TITLE
Markdown list container

### PR DESCRIPTION
**Pull request by Zilula**
Created a container component that makes a call to state/store to grab a list of markdown file`s titles and uses a presentational component to render that list.

### CODE REVIEW

File: src/components/SavedMarkdownList.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 
-export default function MarkdownTitles({ titles }) {
+export function MarkdownTitles({ titles }) {
 
   const listOfTitles = titles.map((title, i)  => {
     return <li key={i}> {title}</li>;
```
Comment (MariahAdams) [inline src/components/SavedMarkdownList.js]:
```suggestion
    return <li key={i}>{title}</li>;
```
Keep that whitespace consistent!

File: src/containers/MarkdownList.js [hunk 2]
```diff
@@ -1,2 +1,33 @@
...
+      }
+    }
+
+    render() {
+      const { markdownTitles } = this.state;
+      return (
+        <MarkdownTitles
+          titles={markdownTitles}
+        />
+
```
Comment (MariahAdams) [inline src/containers/MarkdownList.js]: Extra space pourquoi?

**Review by MariahAdams — COMMENTED** [review]
Looks good, just clean up that whitespace for final approval!

File: src/components/SavedMarkdownList.js [hunk 3]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
```
Comment (acrankin) [inline src/components/SavedMarkdownList.js]:
```suggestion
```
Unnecessary line

**Review by grayson073 — APPROVED** [review]

**Review by caripizza — APPROVED** [review]
🤠

**Pull request merged**